### PR TITLE
Add `test` package to do E2E verification

### PR DESCRIPTION
**Pull request by magodo**
This PR adds a `test` package to do E2E verification. To run the acctest, users need to export the env var that is needed for the AzureRM provider when using the service principal secret auth, together with `AZTFY_E2E` set.

### CODE REVIEW

File: internal/meta/meta_impl.go [hunk 1]
```diff
@@ -5,6 +5,8 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/Azure/aztfy/internal/resmap"
+	"github.com/Azure/aztfy/internal/tfaddr"
```
Comment (ms-henglu) [inline internal/meta/meta_impl.go:9]: The imported packages are not well organized.

File: internal/ui/importlist/importlist.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package importlist
 
 import (
 	"fmt"
+	"github.com/Azure/aztfy/internal/tfaddr"
```
Comment (ms-henglu) [inline internal/ui/importlist/importlist.go:5]: Same here.

File: internal/ui/importlist/importlist_delegate.go [hunk 3]
```diff
@@ -2,9 +2,9 @@ package importlist
 
 import (
 	"fmt"
+	"github.com/Azure/aztfy/internal/tfaddr"
```
Comment (ms-henglu) [inline internal/ui/importlist/importlist_delegate.go:5]: And here.

**Review by ms-henglu — CHANGES_REQUESTED** [review]
Thanks for this PR!
I have left some additional mostly minor comments that once addressed this should be good to merge 👍

**Comment by magodo** [comment]
@ms-henglu Thank you for the comment, I've now ordered the imports with `goimports -w`, please take another look!

**Review by ms-henglu — APPROVED** [review]
LGTM 🎉

**Pull request merged**